pkg/injector/webhook.go

```diff
@@ -182,7 +184,7 @@ func (wh *mutatingWebhook) podCreationHandler(w http.ResponseWriter, req *http.R
 	// We use req.Namespace because pod.Namespace is "" at this point
 	// This string uniquely identifies the pod. Ideally this would be the pod.UID, but this is not available at this point.
 	proxyUUID := uuid.New()
-
+	log.Info().Msg("Creating patches for pod.......")
```

Comment:
Seems like a debug log used during development that can be removed now.

Reply:
yup, fixed!